util_test.go

```diff
@@ -100,6 +100,13 @@ func transactionDiff(expected, got Transaction) map[string]diff {
 					got:      rfB.Interface(),
 				}
 			}
+		} else if rfA.Type() == reflect.TypeOf(&Move{}) {
+			if !rfB.IsNil() && !reflect.DeepEqual(rfA.Interface(), rfB.Interface()) {
+				result[typeField.Name] = diff{
+					expected: rfA.Interface(),
+					got:      rfB.Interface(),
+				}
+			}
```

Comment:
Can we just compare the move id instead? Because really all we want to do when comparing transactions is check to see if the correct move was used. If we want to check PP amounts, we're probably going to check the pokemon itself, not the transaction.

Reply:
I'm just gonna go ahead and make this change. I need the fix in this PR.